src/main/java/org/learntotravel/master/direction/Direction.java

```diff
@@ -0,0 +1,74 @@
+package org.learntotravel.master.direction;
+
+public class Direction {
+
+    private long id;
+
+    private String[] emails;
```

Comment:
In common case (for me for sure) it is more convenient to work with List rather than with arrays. Do we need array for some reason?

Reply:
Fixed, from now it private List<String> emails;